app/components/homePage/GetStarted.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link"
+import ContainerVector from '@/public/assets/images/HomePage/Vector.svg'
+import dynamic from 'next/dynamic'
+
+const LottiePlayer = dynamic(() => import('./LottiePlayer'), {
```

Comment:
If it makes no change can you put this function inside the arrow function for cleaner code